rsmtool/comparer.py

```diff
@@ -329,9 +328,9 @@ def load_rsmtool_output(self, filedir, figdir, experiment_id, prefix, groups_eva
         # read in disattenuated correlations by group
         for group in groups_eval:
             group_dis_corr_file = join(filedir,
-                                     '{}_disattenuated_correlations_by_{}.{}'.format(experiment_id,
-                                                                                     group,
-                                                                                     extension))
+                                       '{}_disattenuated_correlations_by_{}.{}'.format(experiment_id,
+                                                                                       group,
+                                                                                       extension))
```

Comment:
It's a complete nitpick, but why do we use `extension` rather than `file_format` (like elsewhere in the code)?